src/lib/tabs/tab-body.spec.ts

```diff
@@ -153,19 +154,16 @@ describe('MdTabBody', () => {
       fixture = TestBed.createComponent(SimpleTabBodyApp);
     }));
 
-    it('should attach the content when centered and detach when not', fakeAsync(() => {
+    it('should attach when first made visible', fakeAsync(() => {
       fixture.componentInstance.position = 1;
       fixture.detectChanges();
+      tick(1000);
```

Comment:
Leave comment explaining where 1000 came from and why tick was needed

Reply:
Done.